### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by melitus**

### CODE REVIEW

File: packages/foodie-backend/src/loaders/express.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+import express, { Request, Response } from 'express';
+import helmet from 'helmet';
+import AuthRouter from './app/auth/AuthRouter';
+import UserRouter from './app/user/UserRouter';
+// import VendorRouter from './app/vendor/VendorRouter';
+
+
+export default ({ app }: { app: express.Application }) => {
```
Comment (aiveneh) [inline packages/foodie-backend/src/loaders/express.ts]: why this way ?
I think the previous code I much easier to understand

File: packages/foodie-backend/src/middlewares/index.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import attachCurrentUser from './attachCurrentUser';
```
Comment (aiveneh) [inline packages/foodie-backend/src/middlewares/index.ts]: I dint think we should have na index entry point, if youre improrting a file, go to the file and import in directly

File: packages/foodie-backend/src/utils/encryption.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import crypto from 'crypto'
```
Comment (aiveneh) [inline packages/foodie-backend/src/utils/encryption.js]: what are we going to be needing encryption for ?

File: packages/foodie-web/src/selectors/index.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+const getTotalTransactions = createSelector(
+  [ getVisibleTransactions ],
+  (visibleTransactions) => {
+    const withdrawal = getTotal('withdrawal')(visibleTransactions)
+    const deposit = getTotal('deposit')(visibleTransactions)
+    return {withdrawal, deposit}
+  }
+)
+
+export {
```
Comment (aiveneh) [inline packages/foodie-web/src/selectors/index.js]: prefer individual exports

File: packages/foodie-backend/src/loaders/express.ts (hunk 1 above)
Comment (melitus) [reply packages/foodie-backend/src/loaders/express.ts]: Ok, we revert to the one we can both understand. Is just a way to let typescript know that it is an express app

File: packages/foodie-backend/src/middlewares/index.ts (hunk 2 above)
Comment (melitus) [reply packages/foodie-backend/src/middlewares/index.ts]: I will remove this. Thought i removed it when i was doing the folder restructuring

File: packages/foodie-backend/src/utils/encryption.js (hunk 3 above)
Comment (melitus) [reply packages/foodie-backend/src/utils/encryption.js]: For encrypting some important strings text data

File: packages/foodie-web/src/selectors/index.js (hunk 4 above)
Comment (melitus) [reply packages/foodie-web/src/selectors/index.js]: ok, we go with that

**Pull request merged**